src/components/Timeline/Event.js

```diff
@@ -42,22 +46,30 @@ export const Event = ({
         <Card
             className="incident-card"
             style={{ backgroundColor }}
+            expanded={isAllPlaybookInfoAvailable ? null : false}
```

Comment:
How does the expanded property being null differ from false within the view context? Does this change what is displayed? I would much prefer to use something that evals to true or false over something which evals to null or false as nil/null. See also: http://www.yegor256.com/2014/05/13/why-null-is-bad.html

(Specifically, this is confusing as heck for folks who rely on falsiness or truthiness in conditionals, as `!!false == !!null`.) I think collapsible is the antonym of expandable.

Reply:
The MaterialUI Card controls have this weird behavior where you can control expansion manually by passing true or false or let the Card control expansion in local state by passing null. We want to be able to prevent expansion at this level but not explicitly control when things are expanded, so this is valid within the context of how Card handles these arguments. Whether Card is well designed in the first place is a different discussion.

Reply:
![tire fire on a tire swing](http://78.media.tumblr.com/tumblr_me0lp4VpPx1qgcra2o1_500.gif)